ext/curl/tests/curl_basic_010.phpt

```diff
@@ -7,7 +7,7 @@ curl
 --SKIPIF--
 <?php
     $addr = "www.".uniqid().".".uniqid();
```

Comment:
Surely we could try to come up with something that we know does not exist? Like a weird subdomain name of `php.net`